src/main/kotlin/step2/lotto/domain/PlayResults.kt

```diff
@@ -0,0 +1,45 @@
+package step2.lotto.domain
+
+class PlayResults {
+    var elements: MutableList<PlayResult> = mutableListOf()
+        private set
```

Comment:
elements는 MutableList로 선언되어 있기 때문에 var로 만들지 않아도 됩니다. 사용되는 곳이 없다면 private 접근자도 추가해볼까요?
```suggestion
    private val elements: MutableList<PlayResult> = mutableListOf()
        private set
```

Reply:
값이 변경될 수 있는 변수를 선언할 때 무의식적으로 var을 사용한 것 같습니다. 😭

위에서 말씀하셨던 것처럼 가변 객체, var 사용을 최소화 할 수 있도록 수정하겠습니다!